<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphics</class>
 <widget class="QWidget" name="Graphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>485</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="expression_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	border: 1px solid gray;
	background-color: white;
}</string>
     </property>
     <property name="text">
      <string>sin(x)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCustomPlot" name="widget" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_build">
     <property name="text">
      <string>Build graphic</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">QLabel {
	border: 1px solid gray;
}</string>
            </property>
            <property name="text">
             <string>min x</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_minX">
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="styleSheet">
             <string notr="true">QLabel {
	border: 1px solid gray;
}</string>
            </property>
            <property name="text">
             <string>max x</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_maxX">
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">QLabel {
	border: 1px solid gray;
}</string>
            </property>
            <property name="text">
             <string>min y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_minY">
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="styleSheet">
             <string notr="true">QLabel {
	border: 1px solid gray;
}</string>
            </property>
            <property name="text">
             <string>max y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_maxY">
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_step">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px solid gray;
}</string>
          </property>
          <property name="text">
           <string>step</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_step">
          <property name="minimum">
           <double>-1000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
